solver/scheduler.go

```diff
@@ -365,6 +365,8 @@ type pipeFactory struct {
 func (pf *pipeFactory) NewInputRequest(ee Edge, req *edgeRequest) pipe.Receiver {
 	target := pf.s.ef.getEdge(ee)
 	if target == nil {
+		dgst := ee.Vertex.Digest()
```

Comment:
maybe protect against `ee.Vertex = nil` ?

Reply:
(applies to other lines too)

Reply:
actually forget, i've seen that was assumed already in the old code

Reply:
yeah, buildkit makes this guarantee for us fortunately :)